specification/operationalinsights/data-plane/Microsoft.OperationalInsights/stable/v1/OperationalInsights.json

```diff
@@ -128,7 +131,17 @@
       "in": "path",
       "required": true,
       "type": "string",
-      "x-ms-parameter-location": "client"
+      "x-ms-parameter-location": "method"
+    },
+    "workspacesParam": {
+      "name": "workspaces",
+      "description": "Comma separated workspace IDs to include in cross-workspace queries.",
+      "in": "query",
+      "collectionFormat": "csv",
```

Comment:
While you're at a refactoring with breaking changes, have you thought of making this a JSON array instead of a CSV list? I know that goes beyond Swagger changes, but I believe it'd improve the user experience.

Reply:
For GET it's a csv in the query string, for POST it's a JSON array in the body. I think that makes sense?

Reply:
Ah, yeah that totally makes sense! Sorry, I should've looked closer.